src/plugins/saved_objects_management/public/management_section/object_view/components/inspect.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { XJsonLang } from '@kbn/monaco';
+import { omit } from 'lodash';
+import { EuiButtonEmpty, EuiCopy, EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { CodeEditor } from '../../../../../kibana_react/public';
+import { SavedObjectWithMetadata } from '../../../../common';
+
+export interface InspectProps {
+  object: SavedObjectWithMetadata<any>;
+}
+const codeEditorAriaLabel = (title: string) =>
+  i18n.translate('savedObjectsManagement.view.inspectCodeEditorAriaLabel', {
+    defaultMessage: 'inspect { title }',
+    values: {
+      title,
+    },
+  });
+const copyToClipboardLabel = i18n.translate('savedObjectsManagement.view.copyToClipboardLabel', {
+  defaultMessage: 'Copy to clipboard',
+});
+export const Inspect = ({ object }: InspectProps) => {
+  const title = object.meta.title || 'saved object';
+  const objectAsJsonString = JSON.stringify(omit(object, 'meta'), null, 2);
+  return (
+    <EuiFlexGroup direction="column" gutterSize="s">
+      <EuiFlexItem>
+        <div className="eui-textRight">
+          <EuiCopy textToCopy={objectAsJsonString}>
+            {(copy) => (
+              <EuiButtonEmpty
+                aria-label={copyToClipboardLabel}
+                size="s"
+                flush="right"
+                iconType="copyClipboard"
+                onClick={copy}
+              >
+                {copyToClipboardLabel}
+              </EuiButtonEmpty>
+            )}
+          </EuiCopy>
+          <EuiSpacer size="s" />
+        </div>
+        <CodeEditor
+          languageId={XJsonLang.ID}
+          value={objectAsJsonString}
+          aria-label={codeEditorAriaLabel(title)}
+          options={{
+            automaticLayout: false,
+            fontSize: 12,
+            lineNumbers: 'on',
+            minimap: {
+              enabled: false,
+            },
+            overviewRulerBorder: false,
+            readOnly: true,
+            scrollbar: {
+              alwaysConsumeMouseWheel: false,
+            },
+            scrollBeyondLastLine: false,
+            wordWrap: 'on',
+            wrappingIndent: 'indent',
+            renderIndentGuides: false,
```

Comment:
There's an annoying UI issue I haven't been able to figure out with the indent guides. When we do show them, on initial render, the guides are misaligned w.r.t the actual indent:

![initial_render](https://user-images.githubusercontent.com/11909450/134092300-ddcaf71a-22d3-482e-90ce-b65658288330.png)

After a hard refresh, the align properly:
![refreshed](https://user-images.githubusercontent.com/11909450/134092345-8a82e3c0-5e48-449c-b790-7346eb68d97a.png)

I've tried initiating a re-render after the component mounts through setting state but that didn't work either.

I've reached out to the presentation team asking about this and for now went with rather just not rendering them.